impl/doc/is_variant.md

```diff
@@ -33,11 +33,15 @@ The derive in the above example code generates the following code:
 #     Nothing
 # }
 impl <T> Maybe<T>{
+    #[inline]
+    #[must_use]
     pub const fn is_just(&self) -> bool {
```

Comment:
I'm not sure whether we should make an accent on `#[inline]` in docs. Let's leave only `#[must_use]` in docs.